designer/pydesigner.py

```diff
@@ -775,4 +801,4 @@
         # reorder tensor for mrtrix3
         DT, KT = img.tensorReorder(img.tensorType())
         dp.writeNii(DT, img.hdr, op.join(metricpath, 'DT'))
-        dp.writeNii(KT, img.hdr, op.join(metricpath, 'KT'))
+        dp.writeNii(KT, img.hdr, op.join(metricpath, 'KT'))
```

Comment:
Need a newline here

Reply:
What do you mean?

Reply:
Literally add a line; see that red circle? It's unhappy about it stylistically. It's considered bad formatting.

Reply:
It's still unhappy, are you doing this on windows? Do you have CRLF set correctly there?

Reply:
I had to open to file in full to see that circle -_-
Surprised to see Git getting upset over the lack of a line.

Reply:
Of course it is, it's used by the Linux kernel, it adheres to their style guide.

Reply:
TIL. I've made the changes to both files. Apparently it can't have an indentations as well.

Reply:
Fun fact: that style descends from K&R C, which was first published in... 1978! That style isn't going anywhere.